hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestStoreFileInfo.java

```diff
@@ -19,11 +19,17 @@
 
 import static org.junit.Assert.*;
```

Comment:
Avoid start import? Not your fault but I think we can fix this checkstyle issue by the way.